pytest_selenium/safety.py

```diff
@@ -63,8 +63,9 @@ def _skip_sensitive(request, sensitive_url):
     """Skip destructive tests if the environment is considered sensitive"""
     destructive = 'nondestructive' not in request.node.keywords
     if sensitive_url and destructive:
-        pytest.skip(
-            'This test is destructive and the target URL is '
-            'considered a sensitive environment. If this test is '
-            'not destructive, add the \'nondestructive\' marker to '
-            'it. Sensitive URL: {0}'.format(sensitive_url))
+        message = 'This test is destructive and the target URL is '\
+            'considered a sensitive environment. If this test is '\
+            'not destructive, add the \'nondestructive\' marker to '\
+            'it. Sensitive URL: {0}'.format(sensitive_url)
+        print(message)
```

Comment:
@Jenselme Sorry, the module is called `warnings`
https://docs.python.org/2/library/warnings.html